src/components/LicensePicker/style.css

```diff
@@ -0,0 +1,25 @@
+.spdx-picker {
```

Comment:
meta point here, this does css directly rather than scss? That means we'll not get the shared changes and the styling approaches are different. Thoughts on a direction here?

Reply:
Given this could be a shared component, i preferred to use pure css instead of scss.
Maybe in some project users couldn't have implemented that preprocessor.
For our specific case, we can create a specific scss file that handles the specific custom style for this component.
What do you think?